### DISCUSSION TITLE
Support for Zeno R50

### DISCUSSION THREAD

**Discussion by vinicius121** [robotraconteur/robotraconteur#30] [General]
Hi!

I've seen some videos (https://www.youtube.com/watch?v=ACUD_ZGcdnY) of this library working with the Zeno R50 from Robokind through MATLAB. Is this still available?
If so is there any guide/tutorial to help me setup this library with the robot?

Thanks!

**Comment by johnwason** [comment]
Hello @vinicius121 

The drivers for the Zeno R50 were developed by Rensselaer Polytechnic Institute. It doesn't look like they have published the source code. I will follow up with them.

**Comment by vinicius121** [comment]
Thanks, I appreciate it.

**Comment by johnwason** [comment]
We found the driver, but it will need to be cleaned up before it is posted to github. I will let you know when it is ready.

**Comment by vinicius121** [comment]
Ok, thanks. I appreciate it.

**Comment by wlawler45** [comment]
Do you have any information on how to use the robot? We are currently trying to troubleshoot it and are having difficulty finding good documentation on it. Our current problem is that the robot doesn't seem to finish booting.

**Comment by vinicius121** [comment]
There is no much documentation available anymore. The ones that I have are the following links: http://www.robokind.org/basic/gettingStarted.html
http://www.robokind.org/
You can try and plug the robot to a display. On its back it has an HDMI and USB ports. When it boots successfully all the joints should be blocked and the H symbol should remain orange.
Wish I could be more helpful.
